webapp/src/components/form/MarkDownForm.js

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react'
+import * as ReactDOM from 'react-dom'
+import MarkdownIt from 'markdown-it'
+import MdEditor from 'react-markdown-editor-lite'
+
+import 'react-markdown-editor-lite/lib/index.css';
+
+
+const mdParser = new MarkdownIt();
+
+class MarkDownEditor extends React.Component {
+    constructor(props) {
```

Comment:
Nit: The indentation seems inconsistent in this file? Like the constructor is using 4 spaces and the other function 2 spaces?

Reply:
@greeffjean Please fix

Reply:
I actually did this a while ago